planning/behavior_path_dynamic_avoidance_module/include/behavior_path_dynamic_avoidance_module/scene.hpp

```diff
@@ -216,7 +224,7 @@ class DynamicAvoidanceModule : public SceneModuleInterface
 
       // increase counter
       if (counter_map_.count(uuid) != 0) {
-        counter_map_.at(uuid) = std::min(max_count_ + 1, std::max(1, counter_map_.at(uuid) + 1));
+        counter_map_.at(uuid) = std::min(max_count_, counter_map_.at(uuid) + 1);
```

Comment:
The following is better?
IMO, when the counter is negative and the avoidance condition is met, the counter should be moved to 1 quickly. What do you think?
```cpp
std::min(max_count_, std::max(1, counter_map_.at(uuid) + 1));
```

Reply:
>when the counter is negative
I had felt we don't have to care the out of definition range value
I'll follow your suggestion